backend/src/main/java/com/messenger/backend/services/UserService.java

```diff
@@ -13,25 +16,30 @@ public class UserService {
     public UserService(UserRepository userRepository) {
         this.userRepository = userRepository;
     }
+
     public UserEntity getByUserName(String userName) {
         return userRepository.findByUserName(userName);
     }
 
     public Boolean getUserStatus(Integer id) {
-        Boolean status =false;
         UserEntity user = userRepository.findAllById(id);
-        status = user.getActiveStatus();
+        boolean status = user.getActiveStatus();
         return status;
     }
 
+    public UserEntity createUser(String userName) {
+        UserEntity newUser = new UserEntity(null, userName, new Date(), true, Collections.emptySet());
+        return userRepository.save(newUser);
+    }
 
     public UserEntity updateContactList(UserEntity userEntity) {
         Integer id = userEntity.getId();
+        UserEntity result = UserEntity.EMPTY_ENTITY;
         if (userRepository.existsById(id)) {
```

Comment:
Should we test with "existsById(id)" everywhere? For example in getUserStatus method?